wiki/Featured_Artists/Featured_Artist_Showcase_Beatmaps/en.md

```diff
@@ -0,0 +1,359 @@
+- **[Amidst](https://osu.ppy.sh/home/news/2021-03-27-new-featured-artist-amidst)** (2021-03-27)
+  - (![][o!s]) [Amidst - Digital](https://osu.ppy.sh/beatmapsets/1320499#osu/27355601) hosted by ![][flag_RU] [Daycore](https://osu.ppy.sh/users/5596337)
+  - (![][o!s]) [Amidst - Fate](https://osu.ppy.sh/beatmapsets/1359944#osu/2814151) hosted by ![][flag_US] [Aeril](https://osu.ppy.sh/users/4334976)
+  - (![][o!s]) [Amidst - Treat (Cut Ver.)](https://osu.ppy.sh/beatmapsets/1345615#osu/2786730) hosted by ![][flag_RU] [Daycore](https://osu.ppy.sh/users/5596337)
+  - (![][o!s]) [Amidst & OHEY - Need U (Cut Ver.)](https://osu.ppy.sh/beatmapsets/1319384#osu/2733104) hosted by ![][flag_RU] [Shmiklak](https://osu.ppy.sh/users/5504231)
+- **[Xanthochroid](https://osu.ppy.sh/home/news/2021-03-27-new-featured-artist-amidst)** (2021-03-27)
+  - (![][o!s]) [Xanthochroid - The Sound of Hunger Rises](https://osu.ppy.sh/beatmapsets/1335143#osu/2766094) hosted by ![][flag_US] [eiri-](https://osu.ppy.sh/users/3388410)
+
+### April 2021
+
+- **[YUZUKINGDOM](https://osu.ppy.sh/home/news/2021-04-10-new-featured-artist-yuzukingdom)** (2021-04-10)
+  - (![][o!s]) [YUZUKINGDOM - Roskill Lavender](https://osu.ppy.sh/beatmapsets/1410727#osu/2908440) hosted by ![][flag_US] [Nao Tomori](https://osu.ppy.sh/users/5364763)
+
+## History
+
+Even from its very beginning, the Mappers' Guild has been set into pace with "creating promotional beatmap packs for upcoming featured artists" <!-- https://osu.ppy.sh/community/forums/topics/875743 --> as one of its primary goals; in fact, the six beatmaps that were made for the [MOtOLOiD featured artist announcement](https://osu.ppy.sh/home/news/2017-11-07-new-featured-artist-motoloid) were also the Guild's first ever creations. 
+
+As the Guild itself was operating on a closed, application-based entry in its early days, access to upcoming featured artist contents were pften restricted only to a handful of mappers who were predominantly of the [osu!](wiki/Game_mode/osu!) game mode. However, [the Guild's shift to an open and automated entry system in early 2019](wiki/Mapping_and_Modding_Timeline#march.5) has gradually allowed more and more mappers to participate in the creation of showcase beatmaps for future featured artists over time regardless of their game mode given that they either have achieved Tier 1 (or higher) in [Guild rank](/wiki/Mappers_Guild#rewards) or been accepting certain [limited-time mystery quests](wiki/Mappers_Guild#quests) set by the Guild in advance.
+
+Starting from [MYUKKE.'s featured artist announcement](https://osu.ppy.sh/home/news/2020-12-05-new-featured-artist-myukke) in December 2020 onwards, new featured artist announcement posts have also began to feature short preview videos — often with these showcase beatmaps as their main headline — every now and then.
+
+[o!s]: /wiki/shared/mode/osu.png "osu!"
```

Comment:
```suggestion
[o!s]: /wiki/shared/mode/osu.png "osu!standard"
```

Reply:
Not standard. just osu!. it's not hard.

Reply:
> Not standard. just osu!. it's not hard.

The only problem is that `osu!standard` is written as such all over the place, if the writing for the `standard` game mode is indeed that one, we should think about changing it all over the place, on each article.

Reply:
The ASC needs that change as well if so.

Reply:
ah, yeah, i've adjusted the alt texts accordingly in one of my earlier commits dw ;~;

![image](https://user-images.githubusercontent.com/36564236/114387245-67a5ad00-9bbc-11eb-9928-52cfcbdc3f69.png)

![image](https://user-images.githubusercontent.com/36564236/114387343-8015c780-9bbc-11eb-81f0-b65dd57c14ef.png)

as for the asc -- yeah it does seem that the asc needs some updating as it still promotes the use of `osu!standard` despite the fact that `osu!` has been the much preferred term for a long while. will open a separate pr on this in a bit

Reply:
> as for the asc -- yeah it does seem that the asc needs some updating as it still promotes the use of `osu!standard` despite the fact that `osu!` has been the much preferred term for a long while. will open a separate pr on this in a bit

I'm looking into changing `osu!standard` to `osu!` from everywhere, I'll go update at the same time ASC.